scripts/gdal_helper.py

```diff
@@ -53,7 +55,11 @@ def run_gdal(command: List[str], input_file: str = "", output_file: str = "") ->
             gdal_env["AWS_ACCESS_KEY_ID"] = credentials.access_key
             gdal_env["AWS_SECRET_ACCESS_KEY"] = credentials.secret_key
             gdal_env["AWS_SESSION_TOKEN"] = credentials.token
-        command.append(get_vfs_path(input_file))
+            input_file = get_vfs_path(input_file)
+    if input_file_index:
```

Comment:
why do you need this index?

Reply:
Ah, oops. I was trying to copy topo processor and I thought it put the file before the -co args but I just tested it and its fine at the end so I'll remove it.